packages/docs/community/release-process/smart-contracts.md

```diff
@@ -34,26 +29,53 @@ Ongoing smart contract development is done on the `master` branch.
 ### Github tags
 All release branches should be tagged as such, e.g. `celo-contracts-N`. Each should include a summary of the release contents.
```

Comment:
Note that this needs to be updated in light of the tagging approach suggested by @nambrot

Reply:
Will do this in a separate PR